### PR TITLE
Add license snippets in README.md and myst.yml

### LINKED ISSUE #7: Add LICENSE to the repository
Currently the tutorials lack a license. We should choose one and use it here.

We can go with MIT (the same one used in SimPEG) or we can go with a CC-BY, since the code we are sharing here is mostly for educational purposes. Either choice would be good.

We should:
- add a `LICENSE` file to the repo,
- include a License section in the `README.md`,
- set the license in Myst configuration (see https://mystmd.org/guide/frontmatter#licenses)

### PR DESCRIPTION
**Pull request by santisoler**
Add CC-BY and MIT licenses snippets in the `README.md` and configure them for the Myst website in `myst.yml`. Add CC-BY license text in a new `LICENSE-CC-BY` file.

Fixes #7 

For reference, the licenses that can be set in the Myst configuration file are listed here: https://mystmd.org/guide/frontmatter#licenses

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,3 +5,10 @@ Some trouble shooting items for continued development
 
 * Updating MyST: `pip install -U mystmd`
 * After updating, `myst clean --all` to clean cached templates
+
+
+## License
+
+All code and text in this repository is free software: you can redistribute it
+and/or modify it under the terms of the MIT License.
+A copy of this license is provided in [LICENSE](LICENSE).
```
Comment (lheagy) [inline README.md:14]: Should we also mention that content is creative commons (CC-BY-4.0)? There is a lot of text (or maybe we can simply include this in the footer of the website?)

Comment (santisoler) [reply README.md:14]: Totally. I think we should use CC-BY for text and figures, except where explicitly stated.

Does it make sense to use CC-BY as the license in the Myst configuration then?

File: myst.yml [hunk 2]
```diff
@@ -2,8 +2,9 @@
 version: 1
 project:
   title: SimPEG User Tutorials
-  # github: 
-  # description: 
+  license: MIT
```
[the remark this answers is not in the document]
Comment (rowanc1) [reply myst.yml]: You can also do:

```
licence:
   content: CC-BY-4.0
   code: MIT
```

**Comment by santisoler** [comment]
I think it's easier to just use CC-BY for the whole repo, than specifying MIT for code and CC-BY for text and figures.

@dccowan, since you authored most of the content in this repo, do you agree with licensing it under CC-BY? Feel free to comment here or just merge the PR if you agree.

File: myst.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (lheagy) [reply myst.yml]: :+1: This is awesome! I would suggest we go with being explicit about code under MIT and content under CC-BY as that is the most appropriate license for each type of content

File: myst.yml [hunk 3]
```diff
@@ -2,8 +2,9 @@
 version: 1
 project:
   title: SimPEG User Tutorials
-  # github: 
-  # description: 
+  license: CC-BY-4.0
```
Comment (lheagy) [inline myst.yml:5]:
```suggestion
  licence:
   content: CC-BY-4.0
   code: MIT
```

Comment (santisoler) [reply myst.yml:5]: Just noticed that we can set two licenses through Myst. Thanks for the comment @lheagy

**Comment by santisoler** [comment]
I'm merging this. In case we change our minds on the licenses, we can discuss it in future issues/prs.

**Pull request merged**